src/entities/Proposal/routes.ts

```diff
@@ -495,6 +496,17 @@ export async function removeProposal(req: WithAuth<Request<{ proposal: string }>
   return true
 }
 
+export async function proposalComments(req: Request<{ proposal: string }>){
+  const proposal = await getProposal(req)
+  try{
+    const comments = await Discourse.get().getTopic(proposal.discourse_topic_id)
+    return filterComments(comments);
+  } catch (e) {
+    console.log(`Could not get proposal comments, `, e)
```

Comment:
the use of `Logger` is preferred over `console`

```suggestion
    logger.error(`Could not get proposal comments`, e)
```

Reply:
Yeah, I'd rather use a logger but I didn't see it anywhere in the code, so I went with `console` for being consistent with what's already in the router. 

Maybe we should consider creating a separate PR for changing all the `console.log` uses to `logger.error` / `logger.info`?

Reply:
I created a new [ticket in notion](https://www.notion.so/Replace-console-logs-for-logger-8dee70749e23468391180023b980ef83) for replacing the console.log uses with Logger